man/fi_fabric.3.md

```diff
@@ -157,6 +157,17 @@ A fabric identifier.
 
 The name of the underlying fabric provider.
 
+For debugging purposes, environment variables can be used to control which
```

Comment:
Is it always only for debugging purposes?  It might be ok to be a bit more general here, e.g., "For administrative purposes..."